src/components/crossDevice/Intro/index.js

```diff
@@ -27,18 +27,18 @@ const Intro = ({translate, nextStep, mobileConfig}) => {
       <PageTitle
         title={translate(`cross_device.intro.${ isFace ? 'face' : 'document' }.title`)}
       />
-      <div className={classNames(theme.thickWrapper, style.content)}>
+      <ol role="list" className={classNames(theme.thickWrapper, style.content, style.list)}>
       {
         Object.keys(stages).map(key =>
-          <div key={key} className={style.stage}>
+          <li role="listitem" key={key} className={style.stage}>
```

Comment:
is the role necessary if the tag element is used?

Reply:
It is not necessary here. But found VoiceOver was not recognising it as part of a list when parent list element does not have the "list" role despite using the <ul> element HTML. Will need to read more into why this seems to be the case with dynamically generated lists.

Reply:
Looks like chaining Object.keys() with map() is what causes this quirk. Assigning the object keys array returned to a variable first, then mapping it to return the list items resolves it.